# Development and Validation of a Risk Prediction Model of

linezolid-induced thrombocytopenia

Nhi Nguyen Ha (The National DI&ADR Centre, Hanoi University of Pharmacy)  
An Tang Quoc (The National DI&ADR Centre, Hanoi University of Pharmacy)  
Ha Tran Ngan (The National DI&ADR Centre, Hanoi University of Pharmacy)  
Hang Nguyen Thi Thu (The National DI&ADR Centre, Hanoi University of Pharmacy)  
Hoa Vu Dinh (The National DI&ADR Centre, Hanoi University of Pharmacy)  
Nhung TH Trinh (Pharmacoepidemiology and Drug Safety Research Group, Department of Pharmacy, Faculty of Mathematics and Natural Sciences, University of Oslo, Oslo, Norway)  
Anh Nguyen Hoang (The National DI&ADR Centre, Hanoi University of Pharmacy)  
Wednesday, July 24, 2024

Write abstract here, note the indentation

# Checklist

<table style="width:100%;">
<caption>TRIPOD+AI guidance for reporting clinical prediction models that use regression or machine learning methods</caption>
<colgroup>
<col style="width: 9%" />
<col style="width: 1%" />
<col style="width: 85%" />
<col style="width: 3%" />
</colgroup>
<thead>
<tr class="header">
<th><strong>Section/topic</strong></th>
<th style="text-align: center;"><strong>Item</strong></th>
<th><strong>Description</strong></th>
<th style="text-align: center;"><strong>Draft date</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>TITLE</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Title</em></td>
<td style="text-align: center;">1</td>
<td><p><strong>Identify the study as developing or evaluating the performance of a multivariable prediction model, the target population, and the outcome to be predicted</strong></p>
<ul>
<li><p><em>Informative titles aid the identification of prediction model studies by potential readers and also systematic reviewers</em></p></li>
<li><p><em>Report an informative title that provides key information about the target population and the outcome being predicted by the model</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><strong>ABSTRACT</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Abstract</em></td>
<td style="text-align: center;">2</td>
<td><p><strong>See TRIPOD+AI for Abstracts checklist</strong></p>
<ul>
<li><em>Report an abstract addressing each item in the TRIPOD+AI for Abstracts checklist</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><strong>INTRODUCTION</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Background</em></td>
<td style="text-align: center;">3a</td>
<td><p><strong>Explain the healthcare context (including whether diagnostic or prognostic) and rationale for developing or evaluating the prediction model, including references to existing models</strong></p>
<ul>
<li><p><em>Describe the healthcare setting where the model is intended to be used or needed</em></p></li>
<li><p><em>Where an existing prediction model is available, provide a clear justification for developing a new model</em></p></li>
<li><p><em>For studies evaluating an existing model, provide the rationale for the evaluation, and provide references to all models being evaluated</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">3b</td>
<td><p><strong>Describe the target population and the intended purpose of the prediction model in the context of the care pathway, including its intended users (e.g., healthcare professionals, patients, public)</strong></p>
<ul>
<li><p><em>Describe who is the target population for the developed or evaluated model, e.g., people of a certain age, in a specific country, or with a specific disease</em></p></li>
<li><p><em>Describe the intended purpose of the model, including the clinical decision or guidance the model is intended used to support (e.g., referral for further testing or hospital admission, triage, starting a treatment, or changing a lifestyle) and the point in the care pathway the model is to be intended used</em></p></li>
<li><p><em>Describe who the intended users of the model are, and if the model is for healthcare professionals, patients, public or other</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">3c</td>
<td><p><strong>Describe any known health inequalities between sociodemographic groups</strong></p>
<ul>
<li><em>In the context of the healthcare setting where the model is intended to be used, describe any known health inequalities between sociodemographic groups in the target population (along with citations to support the health inequalities)</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Objectives</em></td>
<td style="text-align: center;">4</td>
<td><p><strong>Specify the study objectives, including whether the study describes the development or validation of a prediction model (or both)</strong></p>
<ul>
<li><em>Provide an explicit statement of all objectives of the study, describing whether the study is developing a prediction model, evaluating the performance of a prediction model, or both</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><strong>METHODS</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Data</em></td>
<td style="text-align: center;">5a</td>
<td><p><strong>Describe the sources of data separately for the development and evaluation datasets (e.g., randomised trial, cohort, routine care or registry data), the rationale for using these data, and representativeness of the data</strong></p>
<ul>
<li><p><em>Provide a description of the source of the data used for model development and evaluation of model performance, including whether the data are (for example) from a randomised trial, a cohort, a registry or from electronic routine healthcare records</em></p></li>
<li><p><em>Specify whether the study is using existing data or is prospectively collecting new data for the purpose of the prediction model study</em></p></li>
<li><p><em>Where existing data are being used (i.e., they were originally collected for a different purpose), provide the rationale for using these data, and comment on the suitability (particularly if data are being used from a different setting or country to the intended target population) and representativeness of these data with respect to the intended target population and context</em></p></li>
<li><p><em>A description of the data sources should be provided for all data sets, and separately for development and evaluation</em></p></li>
<li><p><em>If any synthetic data have been used, then provide reasons as to why, and provide all details on how the synthetic data have been created (and code, see item 18f) and used in the study</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">5b</td>
<td><p><strong>Specify the dates of the collected participant data, including start and end of participant accrual; and, if applicable, end of follow-up</strong></p>
<ul>
<li><p><em>Specify the start and end dates of the period for which the participants or the used data were selected</em></p></li>
<li><p><em>For models predicting prognosis, the duration of follow-up is important so report the date of end of follow-up</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Participants</em></td>
<td style="text-align: center;">6a</td>
<td><p><strong>Specify key elements of the study setting (e.g., primary care, secondary care, general population) including the number and location of centres</strong></p>
<ul>
<li><p><em>Describe the healthcare setting, and where the participants in the study were recruited from</em></p></li>
<li><p><em>Report the geographical location (at a minimum, the country) and centres (including the number of centres) of the study</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">6b</td>
<td><p><strong>Describe the eligibility criteria for study participants</strong></p>
<ul>
<li><p><em>The eligibility criteria for participants should be reported to understand the potential applicability and generalisability of the prediction model</em></p></li>
<li><p><em>This includes reporting any restrictions of continuous variables, e.g., age range used to define the eligibility of the included participants</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">6c</td>
<td><p><strong>Give details of any treatments received, and how they were handled during model development or evaluation, if relevant</strong></p>
<ul>
<li><p><em>Any treatments received before or at the start of follow-up should be reported, and whether and how this was handled during the development or evaluation of the prediction model (if relevant)</em></p></li>
<li><p><em>Any treatments received between the moment the prediction model is used and the measurement of the outcome, that could modify the probability of the outcome, should be reported (if relevant)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Data preparation</em></td>
<td style="text-align: center;">7</td>
<td><p><strong>Describe any data pre-processing and quality checking, including whether this was similar across relevant sociodemographic groups</strong></p>
<ul>
<li><p><em>Describe any data cleaning steps, this includes any feature engineering, transformation of raw data, feature reduction and data quality checks. All code used for data cleaning should be made available (see item 18f)</em></p></li>
<li><p><em>For analyses using data from multiple sources (e.g., data from different studies, cohorts, or registries), describe any harmonisation (e.g., of outcome and predictors)</em></p></li>
<li><p><em>Confirm whether all data pre-processing/data cleaning steps were similar across key sociodemographic groups, if relevant</em></p></li>
<li><p><em>If the data pre-processing/data cleaning steps are extensive, consider reporting this information in the supplementary material</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">C-7b</td>
<td><strong>Describe the method for assessing risk of bias and applicability in the individual clusters (eg, using PROBAST)</strong></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Outcomes</em></td>
<td style="text-align: center;">8a</td>
<td><p><strong>Clearly define the outcome that is being predicted and the time horizon, including how and when assessed, the rationale for choosing this outcome, and whether the method of outcome assessment is consistent across sociodemographic groups</strong></p>
<ul>
<li><p><em>For diagnostic prediction models, the outcome should be clearly defined, including whether a (widely accepted) reference standard (ground truth) was used to determine the presence or absence of the outcome</em></p></li>
<li><p><em>For prognostic models, i.e., models predicting an outcome in the future, authors should report the time-horizon of the outcome prediction. For example, predicting the 28-day risk of mortality following cardiothoracic surgery, or the 10-year risk of fractures in patients with osteoporosis. Also, the frequency of outcome assessment during follow-up should be reported</em></p></li>
<li><p><em>If standard definitions are used, e.g., using ICD codes, this should be stated and referenced</em></p></li>
<li><p><em>Any discrepancies in the outcome assessment across socio-demographic groups should be reported</em></p></li>
<li><p><em>In some instances, it may be necessary to confirm that no predictors were used to define the outcome or are a proxy for the outcome</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">8b</td>
<td><p><strong>If outcome assessment requires subjective interpretation, describe the qualifications and demographic characteristics of the outcome assessors</strong></p>
<ul>
<li><p><em>For outcomes that require a subjective interpretation (e.g., interpreting the results from an imaging test, describe the number, qualification, and demographic characteristics of the outcome assessors)</em></p></li>
<li><p><em>If the measurement and interpretation of the outcome require (additional) training or specific instructions, these should be reported.</em></p></li>
<li><p><em>If extensive, consider reporting this information in the supplementary material</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">8c</td>
<td><p><strong>Report any actions to blind assessment of the outcome to be predicted</strong></p>
<ul>
<li><p><em>The outcome being predicted should be assessed blind to information about the predictors – particularly relevant for outcomes requiring a subjective interpretation thereby avoiding data (label) leakage</em></p></li>
<li><p><em>If appropriate, authors should describe which information was available to the outcome assessors and report any specific actions to blinding the outcome assessment</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Predictors</em></td>
<td style="text-align: center;">9a</td>
<td><p>Describe the choice of initial predictors (e.g., literature, previous models, all available predictors) and any pre-selection of predictors before model building</p>
<ul>
<li><p>Provide details on how the initial list of predictors were considered for inclusion in the model building, and whether they were chosen based on a (systematic) review of the literature, clinical input (domain experts), or simply whether using all predictors in the available data</p></li>
<li><p>If any pre-selection of predictors, before model building, was carried out, then provide details how this was done. For example, were predictors omitted for model building due to high amounts of missing data, or predictors not considered plausibly (clinically) related to the outcome being predicted</p></li>
<li><p>The list of initial predictors may be extensive, in these instances reporting these in the supplementary material is advisable</p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">9b</td>
<td><p><strong>Clearly define all predictors, including how and when they were measured (and any actions to blind assessment of predictors for the outcome and other predictors)</strong></p>
<ul>
<li><p><em>All predictors included in the modelling should be clearly defined, along with units of measurement, and all categories for categorical predictors, so that readers and others can replicate, implement, or evaluate the performance of the model</em></p></li>
<li><p><em>Details on how and when the predictor values were measured. Note that predictors should be measured before or at the time the model is intended to be used</em></p></li>
<li><p><em>For predictors requiring subjective interpretation, it may be important to interpret this blind to the values of other predictors considered in the modelling (e.g., avoiding data leakage). Authors should report any actions to blind the assessment of the predictor measurement to other predictors</em></p></li>
<li><p><em>Specifically for diagnostic models, the measurement of the predictors should be done without knowledge of the outcome of the individual as this could artificially inflate the association between the predictors and the outcome. Authors should report any actions to blind the assessment of the predictor measurements to the outcome value</em></p></li>
<li><p><em>In some instances, the number of predictors can be very large and thus reporting them all in the main manuscript is unhelpful, in these instances, it is still important to clearly define all the predictors, and reporting this in the supplementary material should be considered</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">9c</td>
<td><p><strong>If predictor measurement requires subjective interpretation, describe the qualifications and demographic characteristics of the predictor assessors</strong></p>
<ul>
<li><p><em>For predictors that require a subjective interpretation (e.g., interpreting the results from an imaging test), the qualifications and demographic characteristics of the predictor assessors should be reported</em></p></li>
<li><p><em>If the measurement and interpretation require (additional) training or specific instructions, then these should be reported. This could be reported in the supplementary material</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Sample size</em></td>
<td style="text-align: center;">10</td>
<td><p><strong>Explain how the study size was arrived at (separately for development and evaluation), and justify that the study size was sufficient to answer the research question. Include details of any sample size calculation</strong></p>
<ul>
<li><p><em>Describe how the sample size was determined – this should be done separately for determining the sample size needed for model development and the sample size needed to evaluate the performance of the model irrespective of whether data are being prospectively collected or using existing data</em></p></li>
<li><p><em>Provide details and all estimates used in any sample size calculation</em></p></li>
<li><p><em>If no formal sample size calculation was done, e.g., all available data were used, provide a justification whether the size of the data was sufficient to answer the research question</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Missing data</em></td>
<td style="text-align: center;">11</td>
<td><p><strong>Describe how missing data were handled. Provide reasons for omitting any data</strong></p>
<ul>
<li><p><em>Missing data is an omnipresent problem. Authors should report for each predictor being considered for inclusion in the model the number of missing values</em></p></li>
<li><p><em>The handling of missing values should be reported, including any assumptions for the reason of the missingness</em></p></li>
<li><p><em>If individuals (or predictors) have been omitted due to the missing values, this should be reported, and reasons given</em></p></li>
<li><p><em>If missing values have been imputed, then full details of the method for imputing any missing values should be reported</em></p></li>
<li><p><em>If missing values have been imputed confirm it was done separately for the training and any test data (i.e., avoiding leakage)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Analytical methods</em></td>
<td style="text-align: center;">12a</td>
<td><p><strong>Describe how the data were used (e.g., for development and evaluation of model performance) in the analysis, including whether the data were partitioned, considering any sample size requirements</strong></p>
<ul>
<li><p><em>Describe how the available data were used to develop the model and to evaluate model performance, including whether and how the data were partitioned, and the reasons for partitioning the data (e.g., model development, hyperparameter tuning, evaluating model performance, internal-external cross-validation)</em></p></li>
<li><p><em>If the data has been partitioned, report whether sample size requirements (see item 10) were considered during the partitioning, and whether the size of the partitioned data are sufficient to carry out the analyses and answer the research question</em></p></li>
<li><p><em>If the data has been partitioned into training (including any hyperparameter tuning data) and test data, confirm that there has been no data leakage</em></p></li>
<li><p><em>If the data contain multiple records or samples from the same individual, and if the data has been partitioned into training (including any hyperparameter tuning data) and test data, confirm there has been no leakage of individuals across any of the partitioned data or if not, how describe how this was handled in the analysis (see item 12c)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">12b</td>
<td><p><strong>Depending on the type of model, describe how predictors were handled in the analyses (functional form, rescaling, transformation, or any standardisation)</strong></p>
<ul>
<li><p><em>For any predictors that have been transformed during the analysis, i.e., rescaled, or standardised, describe how this was done</em></p></li>
<li><p><em>For any categorical predictors, where collapsing of categories has been carried out, e.g., due to small sample size/too few outcome events, provide the details and reasons</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">12c</td>
<td><p><strong>Specify the type of model, rationale (separately for all model buiding approaches), all model-building steps, including any hyperparameter tuning, and method for internal validation</strong></p>
<ul>
<li><p><em>Clearly specify the type of model (or models) being developed (e.g., logistic regression, Cox regression, random forest, neural network) and provide a rationale for using each model building method – consider the type of outcome being predicted and how the prediction model will be implemented in practice</em></p></li>
<li><p><em>For each model, clearly describe all the steps in the model building, including any hyperparameter tuning, what hyperparameters have been tuned and how this was done. If many model building approaches are being applied and word limits prohibit a full description, then use supplementary material to provide the details</em></p></li>
<li><p><em>For studies that are developing more than one model (e.g., using different model building methods), clearly describe the criteria to choose which is the model being put forward (if any), see item 12e and item 23 on model performance)</em></p></li>
<li><p><em>The internal validation approach (to evaluate model performance) during model development should be clearly described, e.g., was k-fold cross validation or bootstrapping used. Clarify whether all model building steps (including hyperparameter tuning) was replayed during the method of internal evaluation</em></p></li>
<li><p><em>Clearly describe any methods (e.g., bootstrapping) used to examine model stability (e.g., in terms of predictor selection, predictive performance and individual predictions) (Riley &amp; Collins, Biom J 2023; 65: 2200302 [DOI: 10.1002/bimj.202200302])</em></p></li>
<li><p><em>If the data contain multiple records or samples from the same individual, describe how this was handled in the model building and internal validation (e.g., if k-fold cross-validation was used, confirm if all records/samples for an individual were included in the same fold (e.g., avoiding data leakage)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">12d</td>
<td><p><strong>Describe if and how any heterogeneity in estimates of model parameter values and model performance was handled and quantified across clusters (e.g., hospitals, countries). See TRIPOD-Cluster for additional considerations</strong></p>
<ul>
<li><p><em>If the analysis has accounted for any clustering in the data (e.g., from combining individual participant data from multiple studies, or data clustered by medical centre/hospital, or country) during the model development or evaluation of model performance, the rationale and methods used to account for clustering should be clearly described</em></p></li>
<li><p><em>For specific reporting recommendations for prediction model studies that have accounted for clustering and heterogeneity in model parameter values and performance, authors should consult the TRIPOD-Cluster checklist (Debray et al, BMJ 2023; 380: e071018 [DOI: 10.1136/bmj-2022-071018])</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">12e</td>
<td><p><strong>Specify all measures and plots used (and their rationale) to evaluate model performance (e.g., discrimination, calibration, clinical utility) and, if relevant, to compare multiple models</strong></p>
<ul>
<li><p><em>Report all the measures used to evaluate model performance. It is generally expected that as a minimum, model discrimination and calibration (including calibration plots) are presented</em></p></li>
<li><p><em>If the prediction model is predicting a time-to-event outcome, then clearly describe the measures and methods that have been used to account for the time-to-event nature (i.e., censoring). Similarly, the handling of any competing risks should also be stated (if applicable)</em></p></li>
<li><p><em>For prognostic models, report all time-points at which the model’s predictive performance was evaluated</em></p></li>
<li><p><em>Report the methods used for graphical displays of model performance, such as calibration plots (with smooth calibration curves) and decision curves</em></p></li>
<li><p><em>If multiple models are being compared, i.e., comparing against an existing model or comparing multiple modelling approaches, then the methods used for comparing these models, and the criteria for making any judgements on superior performance should be clearly explained</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">12f</td>
<td><p><strong>Describe any model updating (e.g., recalibration) arising from the model evaluation, either overall or for particular sociodemographic groups or settings</strong></p>
<ul>
<li><em>If the model is updated following the validation, such as recalibration or refitting – whether in the entire cohort or in a specific sociodemographic group, then provide details on the methods used to update the model</em></li>
</ul></td>
<td style="text-align: center;">Inapplicable</td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">12g</td>
<td><p><strong>For model evaluation, describe how the model predictions were calculated (e.g., formula, code, object, application programming interface)</strong></p>
<ul>
<li><p><em>For studies evaluating an existing model in a separate data set (i.e., an external validation study), provide details on how the individual predictions from the model were calculated. If a model is not freely/publicly available, explain how the predictions were obtained</em></p></li>
<li><p><em>If a regression model equation was being evaluated, provide details of this equation (e.g., consider presenting this equation, provide a citation to the original study that developed the equation)</em></p></li>
<li><p><em>For studies evaluating a prediction model where there is no equation (e.g., a neural network, random forest), provide details on how the predictions were made, e.g., code, software object, API, and where can this be found (i.e., URL, DOI)</em></p></li>
<li><p><em>If individual predictions from the model were used to create risk groups or classifications (that were not specified in the model development) then details on how and why this was done should be reported (see item 15)</em></p></li>
</ul></td>
<td style="text-align: center;">Inapplicable</td>
</tr>
<tr class="odd">
<td><em>Class imbalance</em></td>
<td style="text-align: center;">13</td>
<td><p><strong>If class imbalance methods were used, state why and how this was done, and any subsequent methods to recalibrate the model or the model predictions</strong></p>
<ul>
<li><p><em>If class imbalance methods (e.g., under/over sampling, SMOTE6) have been used, then provide a rationale for doing so, and how this was done – considering any impact on sample size (e.g., for undersampling methods)</em></p></li>
<li><p><em>Imbalance corrections have an impact on model calibration (van den Goorbergh et al, J Am Med Inform Assoc 2022; 29: 1525–1534 [DOI: 10.1093/jamia/ocac093]), yielding probability estimates that are too high (which also has an impact on defining any risk groups), describe the methods used to recalibrate the model or the model predictions</em></p></li>
</ul></td>
<td style="text-align: center;">Inapplicable</td>
</tr>
<tr class="even">
<td><em>Fairness</em></td>
<td style="text-align: center;">14</td>
<td><p><strong>Describe any approaches that were used to address model fairness and their rationale</strong></p>
<ul>
<li><p><em>Fairness refers to ensuring that a prediction model does not discriminate against individuals or groups, for example based on personal attributes such as race, gender, age and all approaches used to address fairness should be clearly explained along with their rationale</em></p></li>
<li><p><em>It is important to ensure the data contains representative groups (of the target population) when developing the model and evaluating its performance and researchers should attempt to demonstrate this</em></p></li>
<li><p><em>If the prediction model is developed using data with underrepresented groups or particular groups not included, then evaluation in these groups in representative data is needed to evaluate the model in these groups, as to increase generalisability to more groups of individuals beyond those in the development and evaluation data</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Model output</em></td>
<td style="text-align: center;">15</td>
<td><p><strong>Specify the output of the prediction model (e.g., probabilities, classification). Provide details and rationale for any classification and how the thresholds were identified</strong></p>
<ul>
<li><p><em>Most models output a probability estimate for an individual, whilst some models turn the output into a classification (e.g., into low or high-risk groups), this should be clearly stated. If classification or risk groups have been created, then the rationale for doing so in the context of the care pathway and how these risk groups inform any clinical decisions should be made</em></p></li>
<li><p><em>For models producing a classification or risk groups, this should be clearly reported, and any thresholds (e.g., range of estimated probabilities defining the groups) should be specified (whether these are based on the literature, clinical guidelines, statistical considerations or ad-hoc)</em></p></li>
<li><p><em>If uncertainty intervals for individual prediction model outputs have been presented then provide details on how this was done (e.g., using the variance-covariance matrix of parameter estimates or conformal prediction)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Development versus evaluation</em></td>
<td style="text-align: center;">16</td>
<td><p><strong>Identify any differences between the development and evaluation data in healthcare setting, eligibility criteria, outcome, and predictors</strong></p>
<ul>
<li><em>Prediction models developed in one setting, centre or country are not necessarily useful in a different setting, centre, or country. Eligibility criteria, outcome and predictors definitions might (intentionally) differ between data from different sources. Describing any differences between the development data and data used to evaluate model performance is useful to understand and interpret the performance and generalisability of the model in the context of the original model development data</em></li>
</ul></td>
<td style="text-align: center;">Inapplicable (yet)</td>
</tr>
<tr class="odd">
<td><em>Ethical approval</em></td>
<td style="text-align: center;">17</td>
<td><p><strong>Name the institutional research board or ethics committee that approved the study and describe the participant-informed consent or the ethics committee waiver of informed consent</strong></p>
<ul>
<li><em>If the study has no institutional research board or ethics approval, then clearly state so, with reasons</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><strong>OPEN SCIENCE</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Funding</em></td>
<td style="text-align: center;">18a</td>
<td><p><strong>Give the source of funding and the role of the funders for the present study</strong></p>
<ul>
<li><p><em>Provide details on whether the study was funded and provide any details on the role the funder had in the study.</em></p></li>
<li><p><em>Provide any additional funding sources of all authors</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Conflicts of interest</em></td>
<td style="text-align: center;">18b</td>
<td><p><strong>Declare any conflicts of interest and financial disclosures for all authors</strong></p>
<ul>
<li><em>Disclose any of the authors’ relationships or activities that readers could consider pertinent or that may have influenced the study design, conduct, interpretation, or reporting</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Protocol</em></td>
<td style="text-align: center;">18c</td>
<td><p><strong>Indicate where the study protocol can be accessed or state that a protocol was not prepared</strong></p>
<ul>
<li><p><em>Provide all details on the availability of the study protocol, including where the study protocol can be found (e.g., publication details, in supplementary material, publicly available in a repository such as on the Open Science Framework), including a URL or DOI</em></p></li>
<li><p><em>Clearly state if no study protocol was developed or publicly available (and reasons)</em></p></li>
<li><p><em>If there are any notable deviations from what was specified in the study protocol, provide a summary and reasons for the deviation</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Registration</em></td>
<td style="text-align: center;">18d</td>
<td><p><strong>Provide registration information for the study, including register name and registration number, or state that the study was not registered</strong></p>
<ul>
<li><p><em>If the study has been registered (e.g., on clinicaltrials.gov, Open Science Framework), then provide details on the registration number, the name of the register and a link to the registration (including any DOI)</em></p></li>
<li><p><em>Clearly state if the study has not been registered</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Data sharing</em></td>
<td style="text-align: center;">18e</td>
<td><p><strong>Provide details of the availability of the study data</strong></p>
<ul>
<li><p><em>Provide details on the availability of the study data, including where the data can be found (e.g., public repository, URL, DOI), how it can be retrieved, any conditions or restrictions on obtaining and using the data. A data dictionary should accompany any shared data.</em></p></li>
<li><p><em>If data cannot be shared, provide reasons as to why</em></p></li>
<li><p><em>Avoid platitudes such as ‘Data available upon reasonable request’ without specifying conditions for what constitutes a reasonable request</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Code sharing</em></td>
<td style="text-align: center;">18f</td>
<td><p><strong>Provide details of the availability of the analytical code (e.g., any data cleaning, feature engineering, model building, evaluation)</strong></p>
<ul>
<li><p><em>Provide all details on the availability of the analytical code (and documentation on how to run the code), including where the code can be found (e.g., code repository, DOI, link), how it can be retrieved, any conditions or licences to obtain and use the code should be reported (and version)</em></p></li>
<li><p><em>The analytical code is all code needed to replicate (in principle) all the reported results and findings of the study (including any code for data cleaning). The software and any packages needed to reproduce (in principle) the study findings should be reported (including any version numbers). In some instances, more details on the computing environment may need to be reported (e.g., hardware, operating system, CPU, RAM)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><strong>PATIENT &amp; PUBLIC INVOLVEMENT</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Patient &amp; Public Involvement</em></td>
<td style="text-align: center;">19</td>
<td><p><strong>Provide details of any patient and public involvement during the design, conduct, reporting, interpretation, or dissemination of the study or state no involvement</strong></p>
<ul>
<li><p><em>Describe how patients or public were involved in the planning, design, conduct, reporting or dissemination of the study and its findings.</em></p></li>
<li><p><em>Were the findings of the study presented to patients or the public?</em></p></li>
<li><p><em>Consider using the GRIPP2 statement to report patient and public involvement in the research (Staniszewska et al, BMJ 2017; j3453 [DOI: 10.1136/bmj.j3453])</em></p></li>
<li><p><em>If no patients or public were involved in any aspect of the study, then clearly state so</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><strong>RESULTS</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Participants</em></td>
<td style="text-align: center;">20a</td>
<td><p><strong>Describe the flow of participants through the study, including the number of participants with and without the outcome and, if applicable, a summary of the follow-up time. A diagram may be helpful</strong></p>
<ul>
<li><p><em>A flow diagram can be useful to describe the flow of participants through a study, where the entry point to the flow diagram is the source of participants, and then successive steps can relate to eligibility criteria, follow-up (if applicable). and data availability</em></p></li>
<li><p><em>Other useful information to present in the flow diagram include the number of participants with missing values, and the number of outcome events</em></p></li>
<li><p><em>For studies of prognosis or diagnosis with delayed reference testing, a summary of the follow-up time should be reported (e.g., median follow-up, and range)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">20b</td>
<td><p><strong>Report the characteristics overall and, where applicable, for each data source or setting, including the key dates, key predictors (including demographics), treatments received, sample size, number of outcome events, follow-up time, and amount of missing data. A table may be helpful. Report any differences across key demographic groups</strong></p>
<ul>
<li><p><em>Report, possibly using a table, a summary of all data sets used, including the distribution of outcomes, predictors (e.g., mean/median, standard deviation/interquartile range, frequency), any treatments received, the sample size (and number of outcome events, summary of the follow-up time, and for each predictor, the number and proportion of missing values</em></p></li>
<li><p><em>If relevant, it may be useful to report any differences across key demographic groups of interest</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">20c</td>
<td><p><strong>For model evaluation, show a comparison with the development data of the distribution of important predictors (demographics, predictors, and outcome).</strong></p>
<ul>
<li><em>For studies evaluating the performance of an existing model (including those within a model development study) provide a comparison of the distribution of important variables (e.g., mean/median, standard deviation/interquartile range, frequency), such as demographics, predictors in the model, and outcome, including proportion of missing values. This is probably best presented in a table and consider reporting this by outcome status</em></li>
</ul></td>
<td style="text-align: center;">Inapplicable</td>
</tr>
<tr class="odd">
<td><em>Risk of bias</em></td>
<td style="text-align: center;">C-11</td>
<td><strong>Report the results of the risk-of-bias assessment in the individual clusters</strong></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Model development</em></td>
<td style="text-align: center;">21</td>
<td><p><strong>Specify the number of participants and outcome events in each analysis (e.g., for model development, hyperparameter tuning, model evaluation)</strong></p>
<ul>
<li><p><em>The sample size (including the number of outcome events) should be reported for each analysis (i.e., each model development, each model evaluation), as they can often vary across different analyses in a prediction model study (e.g., after data partitioning, model hyperparameter tuning), and particularly in the presence of missing data</em></p></li>
<li><p><em>If the data contain multiple samples or records for an individual report also report the number of individuals</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Model specification</em></td>
<td style="text-align: center;">22</td>
<td><p><strong>Provide details of the full prediction model (e.g., formula, code, object, API) to allow predictions in new individuals and to enable third-party evaluation and implementation, including any restrictions to access or re-use (e.g., freely available, proprietary)</strong></p>
<ul>
<li><p><em>The ‘product’ of a prediction model development study is the prediction model. It is therefore important to provide details on the model, and how it can be used to allow predictions for new individuals to be made. For example, provide the equation for a regression model, for models developed using methods where the model cannot be ‘written down’ as an equation, provide details on the availability of code, software objects or API so that others can evaluate this model in their own data, or implement it in daily practice</em></p></li>
<li><p><em>If multiple models have been developed, then provide details on the availability of all models</em></p></li>
<li><p><em>Explain how to use the model to allow others to make predictions in new individuals.</em></p></li>
<li><p><em>Provide details of any hardware requirements, and software (and packages) to enable third-party testing, implementation and monitoring</em></p></li>
<li><p><em>If a model cannot be made publicly available (e.g., for commercial reasons), this should be clearly reported, and any conditions on gaining access to the model to enable predictions to be calculated for new individuals and third-party evaluation should be reported</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Model performance</em></td>
<td style="text-align: center;">23a</td>
<td><p><strong>Report model performance estimates with confidence intervals, including for any key subgroups (e.g., sociodemographic). Consider plots to aid presentation</strong></p>
<ul>
<li><p><em>Estimates of all model performance measures described in item 12e should be presented along with confidence intervals.</em></p></li>
<li><p><em>Report model performance estimates for the overall population and for any key groups (e.g., sex, ethnicity) of interest (e.g., as part of fairness checks) with confidence intervals</em></p></li>
<li><p><em>Use plots to present and aid evaluation, such as calibration plots (with smooth calibration curves and distributions of predicted values) and decision curves</em></p></li>
<li><p><em>Report performance estimates for all evaluations undertaken (e.g., in development data; in evaluation data; from internal validation process, etc), including at each time-point examined (for prognostic models)</em></p></li>
<li><p><em>Report any examinations of model stability, e.g., in terms of performance estimates and variability of individual predictions across models developed in bootstrap samples (Riley &amp; Collins, Biom J 2023; 65: 2200302 [DOI: 10.1002/bimj.202200302])</em></p></li>
<li><p><em>Clearly indicate which data have been used to present each performance estimate</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">23b</td>
<td><p><strong>If examined, report results of any heterogeneity in model performance across clusters. See TRIPOD Cluster for additional details</strong></p>
<ul>
<li><p><em>If the evaluation of model performance has accounted for any clustering in the data (e.g., from combining individual participant data from multiple studies, or data clustered by centre/hospital, or country), the results should be reported, along with confidence intervals (see item 23a)</em></p></li>
<li><p><em>For specific reporting recommendations for prediction model studies that have accounted for clustering and heterogeneity in model performance, authors should consult the TRIPOD-Cluster checklist (Debray et al, BMJ 2023; 380: e071018 [DOI: 10.1136/bmj- 2022-071018])</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Model updating</em></td>
<td style="text-align: center;">24</td>
<td><p><strong>Report the results from any model updating, including the updated model and subsequent performance</strong></p>
<ul>
<li><p><em>If the prediction model has been updated (e.g., recalibrated, re-fit) following the validation, details of the updated prediction model to enable third-party evaluation and implementation, including any restrictions to access or re-use should be reported (see item 22)</em></p></li>
<li><p><em>The performance of the updated model should be reported (see items 23a, potentially 23b)</em></p></li>
</ul></td>
<td style="text-align: center;">Inapplicable</td>
</tr>
<tr class="odd">
<td><strong>DISCUSSION</strong></td>
<td style="text-align: center;"></td>
<td></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Interpretation</em></td>
<td style="text-align: center;">25</td>
<td><p><strong>Give an overall interpretation of the main results, including issues of fairness in the context of the objectives and previous studies</strong></p>
<ul>
<li><p><em>Interpretation of the study results places the findings in context of other evidence. If there are existing models, then discuss the findings in the context of these existing studies</em></p></li>
<li><p><em>For studies evaluating the performance of an existing prediction model, if existing studies have evaluated the performance of the model, then it’s important to discuss and summarise these findings and place them in context</em></p></li>
<li><p><em>Ensure the interpretation of the findings do not go beyond the findings reported from the development and evaluation of the model to prevent overinterpretation or ‘spin’</em></p></li>
<li><p><em>It is useful for the reader to understand how performance of the model in the evaluation data compares to the performance of the model in any other evaluation studies of that model. When the results diverge, possible reasons for the difference in model performance should be discussed</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td><em>Limitations</em></td>
<td style="text-align: center;">26</td>
<td><p><strong>Discuss any limitations of the study (such as a non-representative sample, sample size, overfitting, missing data) and their effects on any biases, statistical uncertainty, and generalizability</strong></p>
<ul>
<li><em>Acknowledgement of limitations is an important aspect of any scientific paper – and can refer to any aspect of the study design, conduct or analysis. Provide a meaningful discussion of the study limitations factoring in any concerns related to representativeness of the data used in the analysis, sample size, overfitting and missing data/data quality</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td><em>Usability of the model in the context of current care</em></td>
<td style="text-align: center;">27a</td>
<td><p><strong>Describe how poor quality or unavailable input data (e.g., predictor values) should be assessed and handled when implementing the prediction model</strong></p>
<ul>
<li><p><em>Authors should comment on how to handle unavailable predictor values at the moment the model is intended to be used as part of the care pathway in daily practice. Any strategies to impute missing values at the moment the model is intended to be used should also be evaluated (and thus mentioned in the Methods and Results)</em></p></li>
<li><p><em>Similarly, at the point of implementation, authors should discuss (if relevant) the handling of poor quality input data (e.g., image resolution, data format)</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td></td>
<td style="text-align: center;">27b</td>
<td><p><strong>Discuss whether users will be required to interact in the handling of the input data or use of the model, and what level of expertise is required of users</strong></p>
<ul>
<li><p><em>Provide details on how users are expected or required to interact with the prediction model for the model to be used as intended, for example any considerations for handling the input data</em></p></li>
<li><p><em>Is any expertise or training needed or required to use the model, handle or collect the input data, and if so, provide details</em></p></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td></td>
<td style="text-align: center;">27c</td>
<td><p><strong>Discuss any next steps for future research, with a specific view to applicability and generalizability of the model</strong></p>
<ul>
<li><em>Are further evaluations of the model needed, e.g., in different populations or subgroups, or is the model ready for evaluation in clinical trials, or implementation as part of the care pathway</em></li>
</ul></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>

TRIPOD+AI guidance for reporting clinical prediction models that use regression or machine learning methods

# Introduction

## Background

### 3a. Explain the healthcare context (including whether diagnostic or prognostic) and rationale for developing or evaluating the prediction model, including references to existing models

**First paragraph:** introduction about linezolid and associated ADR including thrombocytopenia

**Second paragraph:** what is already known in the literature about this association (magnitude and associated factors)

**Third paragraph:** the importance of investigation this association in Vietnamese settings and develop a risk prediction model. Why is this study needed?

This study aimed to develop and validate a risk prediction model of linezolid-induced thrombocytopenia adapted to Vietnamese setting. In addition, we constructed a simplified risk score using this model to enhance the applicability of the prediction rule in clinical practice.

### 3b. Describe the target population and the intended purpose of the prediction model in the context of the care pathway, including its intended users (e.g., healthcare professionals, patients, public)

### 3c. Describe any known health inequalities between sociodemographic groups

## Objectives

### 4. Specify the study objectives, including whether the study describes the development or validation of a prediction model (or both)

# Methods

## Data

### 5a. Describe the sources of data separately for the development and evaluation datasets (e.g., randomised trial, cohort, routine care or registry data), the rationale for using these data, and representativeness of the data

This study used data from three tertiary hospitals in Northern Vietnam: Thanh Nhan Hospital, Bach Mai Hospital, and the National Hospital of Tropical Diseases. Patients hospitalized and treated with linezolid were included. The following patients were excluded: (i) those under 18 years of age; (ii) those treated with linezolid for less than 3 days; (iii) those without any recorded platelet count in the period before or after initiation of linezolid therapy; (iv) those with baseline platelet count of \> 450 x 10<sup>9</sup> cells/L; (v) those with any missing recorded values among the specified predictors. Each patient was included only once per admission and the first linezolid treatment course was evaluated. Included patients were followed up until the end of the linezolid treatment course or discharge date whichever comes first.

### 5b. Specify the dates of the collected participant data, including start and end of participant accrual; and, if applicable, end of follow-up

The data was collected from each hospital in two phases: a pilot phase and an extension phase. In the pilot phase, we requested existing datasets at the hospitals. In the extension phase, additional data was collected prospectively. Data was extracted from the electronic medical records of the hospitals, except for the pilot dataset at Bach Mai Hospital which was extracted from physical records. In order to harmonise different datasets, data was filled out in a paper form and stored in Excel.

The pilot datasets were collected from January 01 to June 30, 2020 at Thanh Nhan Hospital; from November 01 to December 31, 2019 at Bach Mai Hospital; from May 01 to December 31, 2021 at the National Hospital of Tropical Diseases. The extension datasets were collected from September 01, 2022 to March 31, 2023 at Thanh Nhan Hospital; from December 01, 2022 to March 31, 2023 at Bach Mai Hospital; from April 01 to September 31, 2022 at the National Hospital of Tropical Diseases. *(comment: no data of total number of patients admitted to these hospitals during each period)*

The anonymized data were extracted from electronical medical records at each medical institution, except data from Bach Mai Hospital in the pilot phase. Individual ID number were assigned to each patient’s hospital admission.

Ethical approval was obtained from….

## Participants

### 6a. Specify key elements of the study setting (e.g., primary care, secondary care, general population) including the number and location of centres

### 6b. Describe the eligibility criteria for study participants

### 6c. Give details of any treatments received, and how they were handled during model development or evaluation, if relevant

## Data preparation

### 7. Describe any data pre-processing and quality checking, including whether this was similar across relevant sociodemographic groups

### C-7b. Describe the method for assessing risk of bias and applicability in the individual clusters (eg, using PROBAST)

## Outcome

### 8a. Clearly define the outcome that is being predicted and the time horizon, including how and when assessed, the rationale for choosing this outcome, and whether the method of outcome assessment is consistent across sociodemographic groups

The outcome of interest is linezolid-induced thrombocytopenia, defined as (i) a platelet count of \< 112.5 x 10<sup>9</sup> cells/L (75% of the lower limit of normal) for patients with a baseline platelet count in the normal range; (ii) A reduction in platelet count of ≥ 25% from the baseline value for patients with a baseline platelet count of \< 150 x 10<sup>9</sup> cells/L \[@zyvoxpr; @xu_establishment_2023; @kawasuji_proposal_2021\].

Normal platelet count is defined as 150-450 x 10<sup>9</sup> cells/L. Baseline platelet count is defined as the last recorded PLT value before the start of linezolid therapy. Participants are considered to have met the outcome if their platelet count value meets the above criteria at any time during linezolid therapy or within 5 days after the end of therapy.

> **Warning**
>
> Thrombocytopenia may occur within a few days after stopping LZD, when the drug hasn’t been completely eliminated. However, it is unknown exactly how long after stopping LZD can a TP event still be attributed to LZD use. We deemed that any TP events that occur after 5 days of stopping LZD would not be related to LZD use.
>
> Our rationale is that after 5 days (120 hrs), LZD is guaranteed to be completely eliminated in all patients, as the longest t<sub>1/2</sub> is 8.3 ± 2.4 hrs in end-stage renal disease patients, +3 SD would be ~16 hrs, so 120 hrs is \>7 half-lives, therefore in patients with the worst clearance, 99% of them would have 99% of the drug eliminated from their systems. Furthermore, trough LZD concentration (C<sub>min</sub>) has previously been identified as a predictor of LI-TP development, and LI-TP itself is mostly reversible after discontinuation, so we would argue that any TP events that occur after LZD has been eliminated from the system would not be related to LZD use.

### 8b. If outcome assessment requires subjective interpretation, describe the qualifications and demographic characteristics of the outcome assessors

### 8c. Report any actions to blind assessment of the outcome to be predicted

## Predictors

### 9a. Describe the choice of initial predictors (e.g., literature, previous models, all available predictors) and any pre-selection of predictors before model building

### 9b. Clearly define all predictors, including how and when they were measured (and any actions to blind assessment of predictors for the outcome and other predictors)

Predictors will be screened for inclusion in the model if they meet all of the following criteria: (i) has been identified as a risk factor of LI-TP in previous studies; (ii) can be collected or evaluated from the information in the datasets; (iii) for concomitant medications, has drug-induced immune thrombocytopenia as an adverse drug reaction with a frequency of at least \> 1/1000 in the drug label or Micromedex; (iv) has consensus from a clinical expert panel as possibly related to LI-TP development.

The following information was extracted from all records:

-   Patient demographics
-   Clinical department where linezolid was initiated.
-   Co-morbidities
-   Invasive procedures performed
-   Infection type
-   Laboratory results
-   Linezolid route of administration
-   Linezolid dose in milligrams.
-   Linezolid duration, defined as the number of days from the first to the last dose of linezolid.
-   Concomitant medications during linezolid therapy

### 9c. If predictor measurement requires subjective interpretation, describe the qualifications and demographic characteristics of the predictor assessors

## Sample size

### 10. Explain how the study size was arrived at (separately for development and evaluation), and justify that the study size was sufficient to answer the research question. Include details of any sample size calculation

Previous studies developing logistic regression models for LI-TP risk predictions have included 4-6 predictors in their final models \[@liu_analysis_2021; @duan_regression_2022; @qin_development_2021; @xu_establishment_2023\]. We expect to include about as many candidate predictors, based on results from the expert opinion survey and the Bayesian Model Selection algorithm. Some of the candidate predictors might be continuous, which may potentially require non-linear modelling and therefore slightly increase the number of variables.

A general rule of thumb is for at least 10 events be available for each candidate predictor considered in a prediction model @peduzzi1995. We have a total of 816 eligible patients and 264 of those have experienced the outcome. If the number of candidate predictors is 7, we would have 37 events per candidate predictor, which is considerably greater than the minimum number required. Even if the number of parameters screened is 20, we would still have 13 events per candidate predictor.

However, the aforementioned rule of thumb have generated some debate in the literature, with recent results suggesting that event per variable criterion is too simplistic and has no strong relation to the predictive performance of a model. Riley et al @riley2019 proposed a different set of criteria to estimate minimum sample size for models developed using logistic regression, which are the following:

1.  Small optimism in predictor effect estimates, defined as a global shrinkage factor of \>= 0.9.
2.  Small absolute difference of \<= 0.05 in the model’s apparent and adjusted Nagelkerke’s R-squared.
3.  Precise estimation of the overall risk in the population.

Criteria 1 and 2 aims to reduce the potential of overfitting. Criteria 3 aims to ensure the overall risk is estimated precisely.

#### Step 1: Choose the number of candidate predictors of interest for inclusion in the model, and calculate the corresponding number of predictor parameters (p)

Note that one predictor may require two or more parameters. For example, a k-category predictor requires k-1 parameters and a continuous predictor model with a non-linear trend requires more than one parameter to be estimated. Also include any potential interaction terms towards the total p.

When using a predictor selection method, p should be defined as the total number of parameters screened, and not just the subset that are included in the final model.

Assuming maximum total p to be 20.

> **Note**
>
> The value of p is assumed to be no larger than 20 because univariate regression shows there are 20 variables that are significantly correlated with the outcome.

In [ ]:
parameter_number <- 20
sample_size <- 816

#### Step 2: Choose sensible values for R<sup>2</sup><sub>CS_adj</sub> and max(R<sup>2</sup><sub>CS_app</sub>) based on previous studies where R<sup>2</sup><sub>CS</sub> is the Cox-Snell R<sup>2</sup> statistic.

The value of max(R<sup>2</sup><sub>CS_app</sub>) is based on the overall prevalence or overall rate of the outcome in the population of interest. The incidence of LI-TP in patients treated with linezolid was estimated to be 37% in a meta-analysis by Zhao et al @zhao_prediction_2024.

The value of R<sup>2</sup><sub>CS_adj</sub> could be based on that for a previously published model in the same setting and population (with similar outcome definition). However, as previous studies does not provide any information to identify a sensible value of the minimum expected Cox-Snell R<sup>2</sup>, the value R<sup>2</sup><sub>CS_adj</sub> will be assumed to correspond to a R<sup>2</sup><sub>Nagelkerke</sub> of 0.50, as baseline platelet count, a “direct” measurement of the outcome, is likely to be a predictor.

In [ ]:
calculate_R2_CS_adj <- function(n = 100, E, R2_N = 0.15) {
  ln_L_null <- E * log(E / n) + (n - E) * log(1 - E / n)
  max_R2_CS_app <- 1 - exp(2 * ln_L_null / n)
  R2_CS_adj <- R2_N / max_R2_CS_app
  return(list(R2_CS_adj = R2_CS_adj, max_R2_CS_app = max_R2_CS_app))
}

(cox_snell_r2 <- calculate_R2_CS_adj(E = 37)$R2_CS_adj)

[1] 0.2048324

#### Step 3: Criterion 1

Calculate the sample size required to ensure Van Houwelingen’s global shrinkage factor (S<sub>VH</sub>) is close to 1. A value of S<sub>VH</sub> \>= 0.90 is generally recommended, which reflects a small amount of overfitting during model development.

In [ ]:
calculate_sample_size_1 <- function(p, R2_CS_adj, S_VH = 0.90) {
  n <- p / ((S_VH - 1) * log(1 - R2_CS_adj / S_VH))
  return(n)
}

(
  sample_size_criterion_1 <-
    calculate_sample_size_1(
      p = parameter_number,
      R2_CS_adj = cox_snell_r2
    ) |> ceiling()
)

[1] 775

[1] 21

We see that 775 participants are required to meet criterion 1.

#### Step 4: Criterion 2

Calculate the shrinkage factor (S<sub>VH</sub>) required to ensure a small absolute difference of \<= 0.05 in the developed model’s apparent and adjusted Nagelkerke’s R<sup>2</sup>. Then derive the required sample size conditional on this value of S<sub>VH</sub>.

In [ ]:
calculate_sample_size_2 <- function(p, R2_CS_adj, max_R2_CS_app, delta = 0.05) {
  S_VH <- R2_CS_adj / (R2_CS_adj + delta * max_R2_CS_app)
  n <- p / ((S_VH - 1) * log(1 - R2_CS_adj / S_VH))
  return(n)
}

(
  sample_size_criterion_2 <-
    calculate_sample_size_2(
      p = parameter_number,
      R2_CS_adj = cox_snell_r2,
      max_R2_CS_app = max_apparent_cox_snell_r2
    ) |> ceiling()
)

[1] 478

[1] 34

We see that 478 participants are required to meet criterion 2.

#### Step 5: Criterion 3

Calculate the sample size required to ensure a precise estimate of the overall risk in the population. The suggested absolute margin of error is \<= 0.05.

In [ ]:
calculate_sample_size_3 <- function(phi_hat, delta = 0.05) {
  n <- (1.96 / delta)^2 * phi_hat * (1 - phi_hat)
  return(n)
}

(
  sample_size_criterion_3 <-
    calculate_sample_size_3(phi_hat = 0.37) |> ceiling()
)

[1] 359

We see that 359 participants are required to meet criterion 3.

#### Step 6: Final sample size

The required minimum sample size is the maximum value from steps 3 to 5, to ensure that each of criteria 1 to 3 are met.

In [ ]:
(minimum_sample_size <- max(sample_size_criterion_1, sample_size_criterion_2, sample_size_criterion_3))

[1] 775

[1] 21

The final estimate of minimum sample size is 775, therefore our data is sufficient for model development with 20 parameters.

The maximum number of parameters that can be screened is 21.

## Missing data

### 11. Describe how missing data were handled. Provide reasons for omitting any data

## Analytical methods

### 12a. Describe how the data were used (e.g., for development and evaluation of model performance) in the analysis, including whether the data were partitioned, considering any sample size requirements

### 12b. Depending on the type of model, describe how predictors were handled in the analyses (functional form, rescaling, transformation, or any standardisation)

### 12c. Specify the type of model, rationale2, all model-building steps, including any hyperparameter tuning, and method for internal validation

### 12d. Describe if and how any heterogeneity in estimates of model parameter values and model performance was handled and quantified across clusters (e.g., hospitals, countries).

Harmonisation between datasets was mainly done via manually recording data to a standardized form. Data was then entered into an Excel spreadsheet. Data cleaning was done by handling duplicates, checking for missing values and inconsistencies. Multiple linezolid treatment episodes in the same patient were treated as duplicates and only the first episode was included in the analysis. Patients with missing values were excluded from subsequent analyses. Inconsistencies were resolved by referring back to the original records.

Before analysis, the extracted predictors are limited to those that meet criteria (i) to (iii) in the previous section:

-   Patient demographics were limited to age in years, gender, and weight in kilograms.
-   Clinical department was recorded into binary variables: intensive care unit, emergency department, and others.
-   Co-morbidities were recorded into binary variables: hypertension, heart failure, angina, myocardial infarction, cerebral vascular accident, diabetes, chronic obstructive pulmonary disease, cirrhosis, malignancies, and hematological disorders.
-   Invasive procedures were recorded into binary variables: endotracheal intubation, central venous catheter insertion, intermittent hemodialysis, and continuous renal replacement therapy.
-   Infection type was recorded into binary variables: community-acquired pneumonia, hospital-acquired pneumonia, skin and soft tissue infection, central nervous system infection, intra-abdominal infection, urinary tract infection, bone and joint infection, septicemia, and sepsis.
-   Laboratory results were limited to serum creatinine, hemoglobin count, white blood cell count, and platelet count. Creatinine clearance was estimated from serum creatinine using the Cockcroft-Gault equation.
-   Linezolid route of administration was recorded into binary variables: intravenous, oral, and both.
-   Linezolid dose in milligrams.
-   Linezolid duration in days.
-   Concomitant medications were recoded to binary variables: carbapenems, daptomycin, teicoplanin, levofloxacin, ibuprofen, naproxen, heparin, clopidogrel, enoxaparin, eptifibatide, carbamazepine, valproic acid, quetiapine, atezolizumab, pembrolizumab, trastuzumab, tacrolimus, fluorouracil, irinotecan, leucovorin, oxaliplatin, pyrazinamide, and rifampin.

### 12e. Specify all measures and plots used (and their rationale) to evaluate model performance (e.g., discrimination, calibration, clinical utility) and, if relevant, to compare multiple models

### 12f. Describe any model updating (e.g., recalibration) arising from the model evaluation, either overall or for particular sociodemographic groups or settings

Inapplicable for a development study.

### 12g. For model evaluation, describe how the model predictions were calculated (e.g., formula, code, object, application programming interface)

Inapplicable for a development study.

## Class imbalance

### 13. If class imbalance methods were used, state why and how this was done, and any subsequent methods to recalibrate the model or the model predictions

## Fairness

### 14. Describe any approaches that were used to address model fairness and their rationale

## Model output

### 15. Specify the output of the prediction model (e.g., probabilities, classification). Provide details and rationale for any classification and how the thresholds were identified

## Development versus evaluation

### 16. Identify any differences between the development and evaluation data in healthcare setting, eligibility criteria, outcome, and predictors

## Ethical approval

### 17. Name the institutional research board or ethics committee that approved the study and describe the participant-informed consent or the ethics committee waiver of informed consent

# Open Science

## Funding

### 18a. Give the source of funding and the role of the funders for the present study

## Conflicts of interest

### 18b. Declare any conflicts of interest and financial disclosures for all authors

## Protocol

### 18c. Indicate where the study protocol can be accessed or state that a protocol was not prepared

## Registration

### 18d. Provide registration information for the study, including register name and registration number, or state that the study was not registered

## Data sharing

### 18e. Provide details of the availability of the study data

## Code sharing

### 18f. Provide details of the availability of the analytical code

# Patient & Public Involvement

## Patient & Public Involvement

### 19. Provide details of any patient and public involvement during the design, conduct, reporting, interpretation, or dissemination of the study or state no involvement

# Results

## Participants

### 20a. Describe the flow of participants through the study, including the number of participants with and without the outcome and, if applicable, a summary of the follow-up time. A diagram may be helpful

### 20b. Report the characteristics overall and, where applicable, for each data source or setting, including the key dates, key predictors (including demographics), treatments received, sample size, number of outcome events, follow-up time, and amount of missing data. A table may be helpful. Report any differences across key demographic groups

### 20c. For model evaluation, show a comparison with the development data of the distribution of important predictors (demographics, predictors, and outcome).

Inapplicable for this study.

## Risk of bias

### C-11. Report the results of the risk-of-bias assessment in the individual clusters

## Model development

### 21. Specify the number of participants and outcome events in each analysis (e.g., for model development, hyperparameter tuning, model evaluation)

## Model specification

### 22. Provide details of the full prediction model (e.g., formula, code, object, API) to allow predictions in new individuals and to enable third-party evaluation and implementation, including any restrictions to access or re-use (e.g., freely available, proprietary)

## Model performance

### 23a. Report model performance estimates with confidence intervals, including for any key subgroups (e.g., sociodemographic). Consider plots to aid presentation

### 23b. If examined, report results of any heterogeneity in model performance across clusters. See TRIPOD Cluster for additional details

## Model updating

### 24. Report the results from any model updating, including the updated model and subsequent performance

Inapplicable for development studies.

# Discussion

## Interpretation

### 25. Give an overall interpretation of the main results, including issues of fairness in the context of the objectives and previous studies

## Limitations

### 26. Discuss any limitations of the study (such as a non-representative sample, sample size, overfitting, missing data) and their effects on any biases, statistical uncertainty, and generalizability

## Usability of the model in the context of current care

### 27a. Describe how poor quality or unavailable input data (e.g., predictor values) should be assessed and handled when implementing the prediction model

### 27b. Discuss whether users will be required to interact in the handling of the input data or use of the model, and what level of expertise is required of users

### 27c. Discuss any next steps for future research, with a specific view to applicability and generalizability of the model

# References